biochemist-python/chapters/File_Parsing.ipynb

```diff
@@ -0,0 +1,602 @@
+    "\n",
+    "<p>RESOLUTION : 1.90 ANGSTROMS  </p>  \n",
+    "<p>PROTEIN ATOMS : 3766  </p>\n",
+    "<p>HETEROGEN ATOMS : 20  </p>\n",
+    "<p>HETNAM Abbreviation: PGH Full name: PHOSPHOGLYCOLOHYDROXAMIC </p>\n",
+    "    \n",
+    "```{admonition} Solution\n",
+    ":class: dropdown\n",
+    "<p>This is one possible solution for the 7tim.pdb parsing exercise.</p>\n",
+    "    \n",
+    "```python\n",
+    "with open(protein_file,\"r\") as outfile2:\n",
+    "    data2 = outfile2.readlines()\n",
+    "    for line in data2:\n",
+    "        if 'RESOLUTION.' in line:\n",
+    "            RESOLUTION_line = line\n",
+    "            words = RESOLUTION_line.split()\n",
+    "            words[2] = words[2].rstrip('.')  # to remove the . from RESOLUTION.\n",
+    "            words[-1] = words[-1].rstrip('.')\n",
+    "            print(words[2], ':', words[3], words[-1])\n",
+    "        if 'PROTEIN ATOMS' in line:\n",
+    "            PROTEIN_ATOMS_line = line\n",
+    "            words = PROTEIN_ATOMS_line.split()\n",
+    "            print(words[2], words[3], ':', words[5] )\n",
+    "        if 'HETEROGEN ATOMS' in line:\n",
+    "            HETEROGEN_ATOMS_line = line\n",
+    "            words = HETEROGEN_ATOMS_line.split()\n",
+    "            print(words[2], words[3], ':', words[5])\n",
+    "        if 'HETNAM' in line:\n",
+    "            HETNAM_line = line\n",
+    "            words = HETNAM_line.split()\n",
+    "            print('HETNAM Abbreviation:', words[1], 'Full name:', words[2])\n",
+    "```\n",
+    "    \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Searching for a particular line number in your file\n",
+    "There is a lot of other information in the PDB file we might be interested in.  For example, we might want to pull out the sequence of the protein.  If we look through the file in a text editor, we notice that the sequence is given in a series of lines that begin with `SEQRES` \n",
+    "\n",
+    "SEQRES   1 A  247  ALA ARG THR PHE PHE VAL GLY GLY ASN PHE LYS LEU ASN  \n",
+    "\n",
+    "followed by a seriew of lines that contain the full sequence using three letter abbreviations for the amino acids.  In this case, we don't want to pull something out of this line, as we did in our previous example, but we want to know which line of the file this is so that we can then pull the sequence from the next few lines.  \n",
+    "\n",
+    "When you use a for loop, it is easy to have python keep up with the line numbers using the `enumerate` command.  The general syntax is\n",
+    "\n",
+    "`for linenum, line in enumerate(list_name):\n",
```

Comment:
Use three ticks for code block. Both of these are showing up on the same line now.

Reply:
Fixed